src/runtime/composables/defineOgImage.ts

```diff
@@ -7,7 +7,7 @@ import { useRouter, useRuntimeConfig, useServerHead, withSiteUrl } from '#import
 
 export function defineOgImageScreenshot(options: OgImageScreenshotOptions = {}) {
   const router = useRouter()
-  const route = router?.currentRoute?.value?.path || ''
```

Comment:
In my testing only `value` could be undefined, nothing before. But maybe you know more @harlan-zw, otherwise feel free to just resolve this comment.

Reply:
hm nice, this does seem overly aggressive, not sure why I did that